packages/shared/src/features/onboarding/shared/StepHeadline.tsx

```diff
@@ -9,33 +9,35 @@ import {
 } from '../../../components/typography/Typography';
 import type { TypographyProps } from '../../../components/typography/Typography';
 import { sanitizeMessage } from './utils';
+import type { WithClassNameProps } from '../../../components/utilities';
 
 export enum StepHeadlineAlign {
   Left = 'left',
   Center = 'center',
   Right = 'right',
 }
 
-export type StepHeadlineProps = {
+export interface StepHeadlineProps extends WithClassNameProps {
```

Comment:
One of the reasons why it is preferred to use interface as much as possible is that if we start to try to add an object `className` to support multiple classes, typescript will complain that there is a conflict with the props and must be resolved.

Reply:
Usually you then use `Omit` or `Pick`, but I am cool with people using `interface`, it is not really an issue since we can mix and match them